dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/provisioning/ActivationType.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.kernel.feature.provisioning;
+
+/**
+ *
+ */
+public enum ActivationType {
+    parallel,
```

Comment:
I'd expect these values to be all upper case.

Reply:
Changed to upper case here.